### ISSUE TITLE
Country

### ISSUE THREAD

**Issue by adiweps** [itchannel/fordpass-ha#417]
Are you able to add South Africa as an option?

**Comment by itchannel** [comment]
Technically it is supported I just haven't finish adding it to the UI yet. 

Where it asks for an email instead put your mobile excluding the 0 and it should work for you. Choose America or europe as the region.

**Comment by SquidBytes** [comment]
#400

**Comment by adiweps** [comment]
Nope, wont let me in with that. Will wait for the addon... thanx, looking forward to it!!

**Comment by JanGoffing** [comment]
What country should we select when signing in? It seems to work with both Australia and UK/Europe, but the door unlock doesn't seem to work.

**Comment by angryshot** [comment]
i'd love Thailand if possible, haven't managed to log in selecting any of the existing options
